test-common/src/main/java/com/box/l10n/mojito/test/XliffUtils.java

```diff
@@ -31,7 +31,7 @@ public static Function<String, String> replaceCreatedDateFunction() {
 
       @Override
       public String apply(String input) {
-        return input.replaceAll("\"createdDate\":\\d+,?", "").replaceAll(",\\}", "}");
+        return input.replaceAll("\"createdDate\":\\d+\\.\\d+,?", "").replaceAll(",\\}", "}");
```

Comment:
Why is this needed? Are the XLIFF output changed?

Reply:
How can we make the output backward compatible? the XLIFF produced should not be changed by this migration